### ISSUE TITLE
Exception when Getting All Devices

### ISSUE THREAD

**Issue by dsully** [senorshaun/homebridge-radiora2#25]
```
Nov 15 16:06:22 server homebridge[2155523]: [11/15/2022, 4:06:22 PM] Homebridge UI] [homebridge-radiora2] Incoming Request: /get-all-devices
Nov 15 16:06:51 server homebridge[2155523]: [11/15/2022, 4:06:51 PM] [Homebridge UI] [homebridge-radiora2] ReferenceError: lights is not defined
Nov 15 16:06:51 server homebridge[2155523]:     at UiServer.processOutput (/usr/lib/node_modules/homebridge-radiora2/homebridge-ui/server.js:62:11)
Nov 15 16:06:51 server homebridge[2155523]:     at /usr/lib/node_modules/homebridge-radiora2/homebridge-ui/server.js:47:59
Nov 15 16:06:51 server homebridge[2155523]:     at Array.map (<anonymous>)
Nov 15 16:06:51 server homebridge[2155523]:     at UiServer.processOutputs (/usr/lib/node_modules/homebridge-radiora2/homebridge-ui/server.js:47:40)
Nov 15 16:06:51 server homebridge[2155523]:     at /usr/lib/node_modules/homebridge-radiora2/homebridge-ui/server.js:34:60
Nov 15 16:06:51 server homebridge[2155523]:     at Array.map (<anonymous>)
Nov 15 16:06:51 server homebridge[2155523]:     at UiServer.processArea (/usr/lib/node_modules/homebridge-radiora2/homebridge-ui/server.js:34:40)
Nov 15 16:06:51 server homebridge[2155523]:     at processTicksAndRejections (node:internal/process/task_queues:96:5)
Nov 15 16:06:51 server homebridge[2155523]:     at async Promise.all (index 0)
Nov 15 16:06:51 server homebridge[2155523]:     at async UiServer.processAreas (/usr/lib/node_modules/homebridge-radiora2/homebridge-ui/server.js:25:7)
```

This causes the button to spin and eventually fail. Devices obviously don't get populated.

**Comment by senorshaun** [comment]
Silly me. Simple fix, I’m just not around my computer today to push it out In the /homebridge-ui/server.js file, line 62 needs to change from lights.push(thing);To this.lights.push(thing);On Nov 15, 2022, at 6:09 PM, Dan Sully ***@***.***> wrote:﻿
Nov 15 16:06:22 server homebridge[2155523]: [11/15/2022, 4:06:22 PM] Homebridge UI] [homebridge-radiora2] Incoming Request: /get-all-devices
Nov 15 16:06:51 server homebridge[2155523]: [11/15/2022, 4:06:51 PM] [Homebridge UI] [homebridge-radiora2] ReferenceError: lights is not defined
Nov 15 16:06:51 server homebridge[2155523]:     at UiServer.processOutput (/usr/lib/node_modules/homebridge-radiora2/homebridge-ui/server.js:62:11)
Nov 15 16:06:51 server homebridge[2155523]:     at /usr/lib/node_modules/homebridge-radiora2/homebridge-ui/server.js:47:59
Nov 15 16:06:51 server homebridge[2155523]:     at Array.map (<anonymous>)
Nov 15 16:06:51 server homebridge[2155523]:     at UiServer.processOutputs (/usr/lib/node_modules/homebridge-radiora2/homebridge-ui/server.js:47:40)
Nov 15 16:06:51 server homebridge[2155523]:     at /usr/lib/node_modules/homebridge-radiora2/homebridge-ui/server.js:34:60
Nov 15 16:06:51 server homebridge[2155523]:     at Array.map (<anonymous>)
Nov 15 16:06:51 server homebridge[2155523]:     at UiServer.processArea (/usr/lib/node_modules/homebridge-radiora2/homebridge-ui/server.js:34:40)
Nov 15 16:06:51 server homebridge[2155523]:     at processTicksAndRejections (node:internal/process/task_queues:96:5)
Nov 15 16:06:51 server homebridge[2155523]:     at async Promise.all (index 0)
Nov 15 16:06:51 server homebridge[2155523]:     at async UiServer.processAreas (/usr/lib/node_modules/homebridge-radiora2/homebridge-ui/server.js:25:7)

This causes the button to spin and eventually fail. Devices obviously don't get populated.

**Comment by dsully** [comment]
Thanks. That worked. I'm having a "No response" issue with some keypads, but I'll open a separate issue.

**Comment by senorshaun** [comment]
fixed with 1.5.2 release
